apps/client/src/services/generated/models/DashboardWidget.ts

```diff
@@ -3,13 +3,20 @@
 /* eslint-disable */
 
 export type DashboardWidget = {
-  title: string;
   type:
-    | 'line'
-    | 'scatter'
-    | 'bar'
-    | 'status-grid'
-    | 'status-timeline'
+    | 'line-chart'
```

Comment:
question: should this type field come from the dashboard package?

Reply:
If we have a good way to import the types into Core, yes. I don't think we should rely on two types.